src/NzbDrone.Core/Notifications/Simkl/SimklService.cs

```diff
@@ -0,0 +1,267 @@
+                    break;
+            }
+
+            return simklResolution;
+        }
+
+        private string MapAudio(MovieFile movieFile)
+        {
+            var simklAudioFormat = string.Empty;
+
+            var audioCodec = movieFile.MediaInfo != null ? MediaInfoFormatter.FormatAudioCodec(movieFile.MediaInfo, movieFile.SceneName) : string.Empty;
+
+            switch (audioCodec)
+            {
+                case "AC3":
+                    simklAudioFormat = "dolby_digital";
+                    break;
+                case "EAC3":
+                    simklAudioFormat = "dolby_digital_plus";
+                    break;
+                case "TrueHD":
+                    simklAudioFormat = "dolby_truehd";
+                    break;
+                case "EAC3 Atmos":
+                    simklAudioFormat = "dolby_digital_plus_atmos";
+                    break;
+                case "TrueHD Atmos":
+                    simklAudioFormat = "dolby_atmos";
+                    break;
+                case "DTS":
+                case "DTS-ES":
+                    simklAudioFormat = "dts";
+                    break;
+                case "DTS-HD MA":
+                    simklAudioFormat = "dts_ma";
+                    break;
+                case "DTS-HD HRA":
+                    simklAudioFormat = "dts_hr";
+                    break;
+                case "DTS-X":
+                    simklAudioFormat = "dts_x";
+                    break;
+                case "MP3":
+                    simklAudioFormat = "mp3";
+                    break;
+                case "MP2":
+                    simklAudioFormat = "mp2";
+                    break;
+                case "Vorbis":
+                    simklAudioFormat = "ogg";
+                    break;
+                case "WMA":
+                    simklAudioFormat = "wma";
+                    break;
+                case "AAC":
+                    simklAudioFormat = "aac";
+                    break;
+                case "PCM":
+                    simklAudioFormat = "lpcm";
+                    break;
+                case "FLAC":
+                    simklAudioFormat = "flac";
+                    break;
+                case "Opus":
+                    simklAudioFormat = "ogg_opus";
+                    break;
+            }
+
+            return simklAudioFormat;
+        }
+
+        private string MapAudioChannels(MovieFile movieFile, string audioFormat)
+        {
+            var audioChannels = movieFile.MediaInfo != null ? MediaInfoFormatter.FormatAudioChannels(movieFile.MediaInfo).ToString("0.0") : string.Empty;
+
+            // Map cases where Radarr doesn't handle MI correctly, can purge once mediainfo handling is improved
+            if (audioChannels == "8.0")
+            {
+                audioChannels = "7.1";
+            }
+            else if (audioChannels == "6.0" && audioFormat == "dts_ma")
```

Comment:
Alot of this was fixed in the mediainfo implementation, you may want to sync back up with the trakt file